frontend/src/reports/VariableDisparityReport.tsx

```diff
@@ -61,22 +61,54 @@ export function VariableDisparityReport(props: VariableDisparityReportProps) {
     : [];
 
   return (
-    <>
+    <Grid container xs={12} spacing={1} justify="center">
+      {!props.hidePopulationCard && (
+        <Grid item xs={12}>
+          <PopulationCard fips={props.fips} />
+        </Grid>
+      )}
+
       {!variableConfig && (
-        <Grid container xs={12} spacing={1} justify="center">
-          <Grid item xs={5}>
-            <Alert severity="error">Data not currently available</Alert>
-          </Grid>
+        <Grid item xs={5}>
+          <Alert style={{ margin: "20px" }} severity="error">
+            This data is not currently available from our ingested datasets, but
```

Comment:
I think the word "ingested" is a bit too technical for users. I'd suggest replacing "our ingested datasets" with "in the Health Equity Tracker". Maybe double check with Chi though